ibis_bigquery/compiler.py

```diff
@@ -419,7 +419,7 @@ def _formatter(translator, expr):
             'TIMESTAMP_ADD', {'h', 'm', 's', 'ms', 'us'}
         ),
         ops.TimestampSub: _timestamp_op(
-            'TIMESTAMP_DIFF', {'h', 'm', 's', 'ms', 'us'}
+            'TIMESTAMP_SUB', {'h', 'm', 's', 'ms', 'us'}
```

Comment:
I can confirm that this is the correct function for TimestampSub

https://github.com/ibis-project/ibis/blob/3f9c6f7be5e547117f897df7e10cc519ffb831f4/ibis/expr/operations.py#L2820-L2827

It appears there is a TimestampDiff operator as well, so we might as well add support for that while we're here.

Reply:
Or actually, I can do that in a follow-up PR